Xamarin.Forms.Xaml/XamlLoader.cs

```diff
@@ -317,6 +317,6 @@ public struct CallbackTypeInfo
 
 		internal static Func<IList<FallbackTypeInfo>, Type, Type> FallbackTypeResolver { get; set; }
 		internal static Action<CallbackTypeInfo, object> ValueCreatedCallback  { get; set; }
-		internal static Func<CallbackTypeInfo, object> InstantiationFailedCallback { get; set; }
+		internal static Func<CallbackTypeInfo, Type, Exception, object> InstantiationFailedCallback { get; set; }
```

Comment:
Instead of adding parameters, can we instead define a new type for our parameter (say `InstantiationFailedInfo`) that has the same properties as `CallbackTypeInfo`, plus `Type` and `Exception`?

That way, we can add more info later if we find we need it without breaking our existing reflection handling.

Reply:
I think it's fine

Reply:
👍